### ISSUE TITLE
Link between recipes

### ISSUE THREAD

**Issue by Morglaf** [deathau/cooklang-obsidian#17]
It would be very useful to be able to make a link between the recipes, for example a tart recipe with a puff pastry ingredient that leads to a puff pastry recipe...
I hope that my proposal is not stupid because it is already integrated...

In any case thank you for your work!

**Comment by pelted** [comment]
I agree, but I believe it needs to be implemented by cooklang first. It is a feature that has been brought up there and would work well in Obsidian.

**Comment by dubadub** [comment]
Done, now supports ingredients that are references to other recipes (for examples sauces)
